assets/js/googlesitekit/modules/datastore/sharing-settings.test.js

```diff
@@ -282,6 +282,25 @@ describe( 'core/modules sharing-settings', () => {
 				);
 			} );
 		} );
+
+		describe( 'rollbackSharingSettings', () => {
+			it( 'dispatches a request to rollback the sharing settings to the currently saved values', () => {
```

Comment:
This is slightly inaccurate as it doesn't dispatch a request, the action itself is a pure operation on the current state.

```suggestion
			it( 'sets the sharing settings to the current saved values', () => {
```